util/constants.py

```diff
@@ -33,3 +35,7 @@
 
 # filled in by oceanutil.py
 CONTRACTS: dict = {}  # [chainID][contract_label] : contract_object
+
+# predictoor
+MIN_PREDICTIONS = 1818
+DEPLOYER_ADDRS: List[str] = []
```

Comment:
I don't quite understand this pattern. I see we have something similar for CONTRACTS but these are not constants. Are they supposed to be global variables? Configs?